pkg/apis/operator/v1alpha1/tektoninstallerset_lifecycle.go

```diff
@@ -29,6 +29,7 @@ const (
 	WebhookReady         apis.ConditionType = "WebhooksReady"
 	ControllerReady      apis.ConditionType = "ControllersReady"
 	AllDeploymentsReady  apis.ConditionType = "AllDeploymentsReady"
+	JobCompleted         apis.ConditionType = "JobCompleted"
```

Comment:
im not sure whether this fits in here.
may be we should add this to TektonHub status. 

If the "Job"we are talking about here is the db-migration, then it will make more sense if it is a status in TektonHub with a proper "DatabaseMigration" (True)

Having a "JobCompleted" status in TektonInstallerSet Status is not much informational in this scenario.

Feel free to create an additional InstallerSet to handle the Job alone. ie, the Job alone could have a different lifecycle.

Reply:
May be PostReconcile might be the place where we can handle the "Job"better

Reply:
+1 on not adding this as installerSet cond